### PR TITLE
en, zh: add some clarify for dashboard

### PR DESCRIPTION
**Pull request by july2993**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
add some clarify for dashboard

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by july2993** [comment]
@DanielZhangQD PTAL

File: zh/access-dashboard.md [hunk 1]
```diff
@@ -70,6 +74,10 @@ kubectl port-forward svc/basic-discovery -n ${namespace} 10262:10262
 
 然后在浏览器中访问 <http://localhost:10262/dashboard> 即可访问到 TiDB Dashboard。
 
+> **注意：**
+>
+> `port-forward` 默认绑定 IP 127.0.0.1, 如果你需要在其它机器访问跑 `port-forward` 命令的机器， 可以添加 --address 指定绑定 IP 地址。
```
Comment (csuzhangxc) [inline zh/access-dashboard.md:79]:
```suggestion
> `port-forward` 默认绑定 IP 127.0.0.1, 如果你需要在其它机器访问跑 `port-forward` 命令的机器， 可以添加 `--address` 指定绑定 IP 地址。
```

这个 `跑` 要换成 `运行` 吗？

File: zh/access-dashboard.md [hunk 2]
```diff
@@ -70,6 +74,10 @@ kubectl port-forward svc/basic-discovery -n ${namespace} 10262:10262
 
 然后在浏览器中访问 <http://localhost:10262/dashboard> 即可访问到 TiDB Dashboard。
 
+> **注意：**
+>
+> `port-forward` 默认绑定 IP 127.0.0.1, 如果你需要在其它机器访问跑 `port-forward` 命令的机器， 可以添加 `--address` 指定绑定 IP 地址。
```
Comment (july2993) [inline zh/access-dashboard.md:79]:
```suggestion
> `port-forward` 默认绑定 IP 127.0.0.1, 如果你需要在其它机器访问运行 `port-forward` 命令的机器， 可以添加 `--address` 指定绑定 IP 地址。
```

File: zh/access-dashboard.md (hunk 1 above)
Comment (july2993) [reply zh/access-dashboard.md]: 已换

File: en/access-dashboard.md [hunk 3]
```diff
@@ -76,6 +80,10 @@ TiDB Dashboard is built in the PD component in TiDB 4.0 and later versions. You
 
 3. Visit <http://localhost:10262/dashboard> in your browser to access TiDB Dashboard.
 
+> **Warning:**
+>
+> `port-forward` default bind IP 127.0.0.1, if you want to access the machine runs `port-forward`, you can specify --address flag to bind the IP address.
```
Comment (csuzhangxc) [inline en/access-dashboard.md:85]:
```suggestion
> `port-forward` default bind IP 127.0.0.1, if you want to access the machine runs `port-forward`, you can specify `--address` flag to bind the IP address.
```

**Review by csuzhangxc — APPROVED** [review]
LGTM

**Comment by qiancai** [comment]
/retitle en, zh: add some clarify for dashboard

File: zh/access-dashboard.md [hunk 4]
```diff
@@ -8,6 +8,10 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/access-dashboard/']
 
 TiDB Dashboard 是 TiDB 4.0 专门用来帮助观察与诊断整个 TiDB 集群的可视化面板，你可以在 [TiDB Dashboard](https://docs.pingcap.com/zh/tidb/stable/dashboard-intro) 了解详情。本篇文章将介绍如何在 Kubernetes 环境下访问 TiDB Dashboard。
 
+> **注意：**
+>
+> discovery 会 proxy 请求到 PD, 本文档我们直接访问的是 discovery 而不是 PD。
```
Comment (DanielZhangQD) [inline zh/access-dashboard.md:13]:
```suggestion
> Discovery 服务会 proxy 请求到 TiDB Dashboard, 本文档我们通过 Discovery 服务访问 TiDB Dashboard。
```

File: zh/access-dashboard.md [hunk 5]
```diff
@@ -70,6 +74,10 @@ kubectl port-forward svc/basic-discovery -n ${namespace} 10262:10262
 
 然后在浏览器中访问 <http://localhost:10262/dashboard> 即可访问到 TiDB Dashboard。
 
+> **注意：**
+>
+> `port-forward` 默认绑定 IP 127.0.0.1, 如果你需要在其它机器访问运行 `port-forward` 命令的机器， 可以添加 `--address` 指定绑定 IP 地址。
```
Comment (DanielZhangQD) [inline zh/access-dashboard.md:79]:
```suggestion
> `port-forward` 默认绑定 IP 127.0.0.1，如果你需要在其它机器访问运行 `port-forward` 命令的机器，可以通过 `--address` 指定绑定的 IP 地址。
```

File: zh/access-dashboard.md [hunk 6]
```diff
@@ -70,6 +74,10 @@ kubectl port-forward svc/basic-discovery -n ${namespace} 10262:10262
 
 然后在浏览器中访问 <http://localhost:10262/dashboard> 即可访问到 TiDB Dashboard。
 
+> **注意：**
+>
+> `port-forward` 默认绑定 IP 127.0.0.1，如果你需要在其它机器访问运行 `port-forward` 命令的机器，可以通过 `--address` 指定绑定的 IP 地址。
```
Comment (qiancai) [inline zh/access-dashboard.md:79]:
```suggestion
> `port-forward` 默认绑定 IP 地址 127.0.0.1。如果你需要使用其它 IP 地址访问运行 `port-forward` 命令的机器，可以通过 `--address` 选项指定需要绑定的 IP 地址。
```

File: zh/access-dashboard.md [hunk 7]
```diff
@@ -8,6 +8,10 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/access-dashboard/']
 
 TiDB Dashboard 是 TiDB 4.0 专门用来帮助观察与诊断整个 TiDB 集群的可视化面板，你可以在 [TiDB Dashboard](https://docs.pingcap.com/zh/tidb/stable/dashboard-intro) 了解详情。本篇文章将介绍如何在 Kubernetes 环境下访问 TiDB Dashboard。
 
+> **注意：**
+>
+> Discovery 服务会 proxy 请求到 TiDB Dashboard, 本文档我们通过 Discovery 服务访问 TiDB Dashboard。
```
Comment (qiancai) [inline zh/access-dashboard.md:13]: 请问 “Discovery 服务会 proxy 请求到 TiDB Dashboard“ 是什么意思呢，是 “Discovery 服务会发送代理请求到 TiDB Dashboard“ 吗？另外，请问 ”Discovery 服务“ 是运行在 TiDB 集群上的吗？

Comment (july2993) [reply zh/access-dashboard.md:13]: 1, 是？ https://zh.wikipedia.org/wiki/%E5%8F%8D%E5%90%91%E4%BB%A3%E7%90%86
2，不是？ Discovery 是单独组件

Comment (DanielZhangQD) [reply zh/access-dashboard.md:13]:
```suggestion
> TiDB Operator 会为每一个 TiDB 集群启动一个 Discovery 服务，Discovery 服务会为每个 PD Pod 返回相应的启动参数，来辅助 PD 集群启动。另外，Discovery 服务也会代理请求到 TiDB Dashboard，本文档我们通过 Discovery 服务访问 TiDB Dashboard。
```

Comment (DanielZhangQD) [reply zh/access-dashboard.md:13]: @july2993 @qiancai 这样好一些吗？

File: en/access-dashboard.md [hunk 8]
```diff
@@ -76,6 +80,10 @@ TiDB Dashboard is built in the PD component in TiDB 4.0 and later versions. You
 
 3. Visit <http://localhost:10262/dashboard> in your browser to access TiDB Dashboard.
 
+> **Note:**
+>
+> `port-forward` default bind IP 127.0.0.1, if you want to access the machine runs `port-forward`, you can specify `--address` flag to bind the IP address.
```
Comment (qiancai) [inline en/access-dashboard.md:85]:
```suggestion
> By default, `port-forward` binds to the IP address 127.0.0.1. If you need to use another IP address to access the machine running the `port-forward` command, you can add the `-address` option and specify the IP address to be bound.
```

File: zh/access-dashboard.md [hunk 9]
```diff
@@ -8,6 +8,10 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/access-dashboard/']
 
 TiDB Dashboard 是 TiDB 4.0 专门用来帮助观察与诊断整个 TiDB 集群的可视化面板，你可以在 [TiDB Dashboard](https://docs.pingcap.com/zh/tidb/stable/dashboard-intro) 了解详情。本篇文章将介绍如何在 Kubernetes 环境下访问 TiDB Dashboard。
 
+> **注意：**
+>
+> TiDB Operator 会为每一个 TiDB 集群启动一个 Discovery 服务，Discovery 服务会为每个 PD Pod 返回相应的启动参数，来辅助 PD 集群启动。另外，Discovery 服务也会代理请求到 TiDB Dashboard，本文档我们通过 Discovery 服务访问 TiDB Dashboard。
```
Comment (qiancai) [inline zh/access-dashboard.md:13]:
```suggestion
> TiDB Operator 会为每一个 TiDB 集群启动一个 Discovery 服务。Discovery 服务会为每个 PD Pod 返回相应的启动参数，来辅助 PD 集群启动。此外，Discovery 服务也会发送代理请求到 TiDB Dashboard。本文档我们将通过 Discovery 服务访问 TiDB Dashboard。
```

File: en/access-dashboard.md [hunk 10]
```diff
@@ -10,6 +10,10 @@ TiDB Dashboard is a visualized tool in TiDB 4.0 used to monitor and diagnose the
 
 This document describes how to access TiDB Dashboard in Kubernetes.
 
+> **Note:**
+>
+> discovery will proxy the request to PD, in this document we access the discovery instead of PD.
```
Comment (qiancai) [inline en/access-dashboard.md:15]:
```suggestion
> TiDB Operator starts a Discovery service for each TiDB cluster. The Discovery service can return the corresponding startup parameters for each PD Pod to support the startup of the PD cluster. The Discovery service can also send proxy requests to the TiDB Dashboard. In this document, TiDB Dashboard is accessed using the Discovery service.
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM with the Chinese version.

**Pull request merged**